src/extensionFunctions.ts

```diff
@@ -220,15 +220,24 @@ export function findTerminal(name: string) : vscode.Terminal | undefined {
 }
 
 // send a message to a named terminal
-export async function sendTerminalText(name:string, text:string): Promise<void> {
+export async function sendTerminalText(name:string, text:string, sendLF = true): Promise<void> {
 	let terminal : vscode.Terminal | undefined = findTerminal(name);
 	if (!terminal) {
 		terminal = vscode.window.createTerminal(name);
 	}
 	if (terminal) {
-		showAndSendText(terminal, text);
+		showAndSendText(terminal, text, sendLF);
+	}
+	const msg = `Sent terminal ${name} the text ${text}`;
+	if (sendLF) {
+		sendTextToOutputChannel(msg + ` with LF`);
```

Comment:
Optional: I think it woudl be better to explicitly mention that it is Linefeed isntead of LF acronym